.travis.yml

```diff
@@ -18,6 +18,13 @@ matrix:
     - rvm: 2.3
       gemfile: Gemfile.rails42
```

Comment:
Can we axe this `matrix` block while we're here? The rvm/gemfile combinations already do it :)